internal/utils/main.go

```diff
@@ -112,6 +112,37 @@ func CreateSampleResultMeta(successful bool, subOperationCount int) xdr.Transact
 	}
 }
 
+func CreateSampleResultPair(successful bool, subOperationCount int) xdr.TransactionResultPair {
```

Comment:
This is only used for testing right?

Reply:
correct, we have to build a sample `TransactionResultPair` to parse in the unit tests